package/firmware/linux-firmware/patches/201-linux-firmware-update-amd-cpu-ucode.patch

```diff
@@ -0,0 +1,77 @@
+From b250b32ab1d044953af2dc5e790819a7703b7ee6 Mon Sep 17 00:00:00 2001
+From: John Allen <john.allen@amd.com>
+Date: Tue, 18 Jul 2023 23:19:59 +0000
+Subject: linux-firmware: Update AMD cpu microcode
+
+* Update AMD cpu microcode for processor family 19h
+
+Key Name        = AMD Microcode Signing Key (for signing microcode container files only)
+Key ID          = F328AE73
+Key Fingerprint = FC7C 6C50 5DAF CC14 7183 57CA E4BE 5339 F328 AE73
+
+Signed-off-by: John Allen <john.allen@amd.com>
+Signed-off-by: Josh Boyer <jwboyer@kernel.org>
+---
+ WHENCE                                 |   2 +-
+ amd-ucode/README                       |   6 +++---
+ amd-ucode/microcode_amd_fam19h.bin     | Bin 16804 -> 16804 bytes
+ amd-ucode/microcode_amd_fam19h.bin.asc |  16 ++++++++--------
+ 4 files changed, 12 insertions(+), 12 deletions(-)
+
+diff --git a/WHENCE b/WHENCE
+index de23a75e..b92e5299 100644
+--- a/WHENCE
++++ b/WHENCE
+@@ -3912,7 +3912,7 @@ Raw: amd-ucode/microcode_amd_fam17h.bin
+ Version: 2023-04-13
+ File: amd-ucode/microcode_amd_fam19h.bin
+ Raw: amd-ucode/microcode_amd_fam19h.bin
+-Version: 2023-01-31
++Version: 2023-07-18
+ File: amd-ucode/README
+ 
+ License: Redistributable. See LICENSE.amd-ucode for details
+diff --git a/amd-ucode/README b/amd-ucode/README
+index 0d97f910..4308fe2d 100644
+--- a/amd-ucode/README
++++ b/amd-ucode/README
+@@ -36,6 +36,6 @@ Microcode patches in microcode_amd_fam17h.bin:
+   Family=0x17 Model=0x31 Stepping=0x00: Patch=0x08301072 Length=3200 bytes
+ 
+ Microcode patches in microcode_amd_fam19h.bin:
+-  Family=0x19 Model=0x01 Stepping=0x00: Patch=0x0a001078 Length=5568 bytes
+-  Family=0x19 Model=0x01 Stepping=0x01: Patch=0x0a0011ce Length=5568 bytes
+-  Family=0x19 Model=0x01 Stepping=0x02: Patch=0x0a001231 Length=5568 bytes
++  Family=0x19 Model=0x01 Stepping=0x01: Patch=0x0a0011d1 Length=5568 bytes
++  Family=0x19 Model=0x01 Stepping=0x00: Patch=0x0a001079 Length=5568 bytes
++  Family=0x19 Model=0x01 Stepping=0x02: Patch=0x0a001234 Length=5568 bytes
+diff --git a/amd-ucode/microcode_amd_fam19h.bin b/amd-ucode/microcode_amd_fam19h.bin
+index 4bd623e1..50470c3f 100644
+Binary files a/amd-ucode/microcode_amd_fam19h.bin and b/amd-ucode/microcode_amd_fam19h.bin differ
```

Comment:
You are missing the actual file  here. It is binary in git, I think you can not add it to patch which only works with text files.

Reply:
Ah, not sure how to handle bins if that is true.

Reply:
I can add the two .bin files after having applied the backports to `./files/` and have the Makefile overwrite the existing ones.  Is that an acceptable option?